src/view/treeNodes/pullRequestNode.ts

```diff
@@ -74,9 +74,13 @@ export class PRNode extends TreeNode implements vscode.CommentingRangeProvider {
 				return [descriptionNode];
 			}
 
-			await this.pullRequestModel.initializeReviewThreadCache();
-			await this.pullRequestModel.initializePullRequestFileViewState();
-			this._fileChanges = await this.resolveFileChangeNodes();
+			[, , this._fileChanges, ,] = await Promise.all([
```

Comment:
I did a quick bit of testing of this, and I noticed that the little speech bubble character that we use as a file decoration for files with comments was no longer showing with this change. I suggested an edit that fixes the issue. This was a long standing bug that was only just revealed now that these methods are run in parallel. In short, this method was getting called when the comments threads changed, but not when they were originally set: https://github.com/microsoft/vscode-pull-request-github/blob/4c077d1ad6c5f8ad54c6adb0253bce87a6d9655e/src/view/treeDecorationProvider.ts#L12-L19